runner/src/main.rs

```diff
@@ -449,6 +449,22 @@ fn run_ci() -> Step {
                     coverage: false,
                 },
             }),
+            // Package the Hello World application in a Docker image.
+            run_examples(&RunExamples {
```

Comment:
Could this be a separate command that can be run individually? I think that would be helpful when working locally.

Reply:
If you wanted to run just this step, you can do so via: `./scripts/docker_run ./scripts/runner run-examples --example-name=hello_world --run-server=false --run-clients=false --build-docker`

Reply:
The code here basically provides the same arguments, just in the shape of a step inside of the `run-ci` task.

Reply:
You are right. Thanks :)